kyuubi-hive-beeline/src/main/java/org/apache/hive/beeline/KyuubiBeeLine.java

```diff
@@ -80,8 +80,6 @@ public KyuubiBeeLine(boolean isBeeLine) {
   @Override
   void usage() {
     super.usage();
```

Comment:
we can eliminate this override then

Reply:
I mean delete the method `usage`, we don't need to override a method that just calls `super.<method-name>`

Reply:
Ok, got it. I've made this modification and committed it.